tests/handler.test.ts

```diff
@@ -34,10 +37,10 @@ describe('handler function', () => {
 
     bookingEvent.Records[0].body =
```

Comment:
If you update event.json and doubleEvent.json, you could remove this line as well as 65 and 67. You would only need to keep line 49.